torch/csrc/distributed/rpc/init.cpp

```diff
@@ -121,7 +121,14 @@ PyObject* rpc_init(PyObject* /* unused */) {
           .def(
               "get_metrics",
               &RpcAgent::getMetrics,
-              py::call_guard<py::gil_scoped_release>());
+              py::call_guard<py::gil_scoped_release>())
+          .def(
+              "_get_timeout",
```

Comment:
Could we use the existing `get_rpc_timeout` which is on the RPC module? https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/rpc/init.cpp#L690

Reply:
Good  catch! I missed that one.